binfmt/binfmt_execmodule.c

```diff
@@ -150,15 +151,26 @@ int exec_module(FAR const struct binary_s *binp,
         }
     }
 
+  /* Make a copy of the environment here */
+
+  if (envp || (envp = get_environ_ptr()))
```

Comment:
@xiaoxiang781216 should we use `environ` here instead of `get_environ_ptr` in https://github.com/apache/incubator-nuttx/pull/6119 ?